Cargo.toml

```diff
@@ -18,25 +18,30 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
 default = ["std"]
-std = ["bitcoin/std", "bitcoin-internals/std"]
-rand = ["bitcoin/rand"]
-rand-std = ["bitcoin/rand-std"]
-serde = ["actual-serde", "bitcoin/serde", "bitcoin-internals/serde"]
+std = ["bitcoin/std", "bitcoin-internals/std", "miniscript?/std"]
+rand = ["std", "bitcoin/rand-std"]
+serde = ["dep:actual-serde", "bitcoin/serde", "bitcoin-internals/serde"]
 base64 = ["bitcoin/base64"]
-miniscript-std = ["std", "miniscript/std"]
-miniscript-no-std = ["miniscript/no-std"]
+miniscript = ["dep:miniscript", "miniscript?/no-std"]
 
 [dependencies]
-bitcoin = { version = "0.32.2", default-features = false }
+bitcoin = { version = "0.32.6", default-features = false }
 bitcoin-internals = { version = "0.3.0", features = ["alloc"] }
-
-# Do NOT use this feature! Use one of the `miniscript-` features instead.
 miniscript = { version = "12.2.0", default-features = false, optional = true }
-# Do NOT use this as a feature! Use the `serde` feature instead.
-actual-serde = { package = "serde", version = "1.0.103", default-features = false, features = [ "derive", "alloc" ], optional = true }
+actual-serde = { package = "serde", version = "1.0.103", default-features = false, features = ["derive", "alloc"], optional = true }
```

Comment:
Excuse me in advance, I'm in a state today.

If you put random whitespace changes like this in a patch it makes the reviewer have to spend some clock cycles trying to workout why the diff contains these lines, and what exactly changed. As a general principle just throw whitespace changes in a separate patch up front. Thanks